djangoApp/settings.py

```diff
@@ -90,6 +90,18 @@
     }
 }
 
+if os.environ.get('GITHUB_WORKFLOW'):
+    DATABASES = {
+        'default': {
+           'ENGINE': 'django.db.backends.postgresql',
+           'NAME': 'github_actions',
+           'USER': 'postgres',
+           'PASSWORD': 'postgres',
+           'HOST': '127.0.0.1',
+           'PORT': '5432',
+        }
+    }
+
```

Comment:
This duplication can easily be avoided if you use the same `POSTGRES_USER`, `POSTGRES_PASSWORD` and `POSTGRES_DB` you used in the first `DATABASES` definition from line 82.